targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs

```diff
@@ -434,14 +434,19 @@ public class PlayFabApiTest
     @Test
     public void CloudScriptServer()
     {
+        PlayFabServerModels.LoginWithServerCustomIdRequest customIdReq = new PlayFabServerModels.LoginWithServerCustomIdRequest();
+        customIdReq.CreateAccount = true;
+        String serverId = playFabId;
+        customIdReq.ServerCustomId = serverId;
```

Comment:
This should use the buildIdentifier the same way the client tests use.
If the server uses a fixed ID, then multiple simultaneous runs of this test will break, all trying to simultaneously access the same player.